src/components/Timeline/Event/EventCard.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { DateTime } from 'luxon'
+import { Card, CardHeader, CardText } from 'material-ui/Card'
+import * as Icons from 'material-ui/svg-icons'
+import Avatar from 'material-ui/Avatar'
+
+import Playbook from 'components/Timeline/Playbook/Playbook'
+import { LoadTextFromEvent } from 'services/playbookService'
+import timeFormattedToMultipleZones from 'helpers/timeFormattedToMultipleZones'
+
+export const EventCard = ({
+  event,
+  titleText,
+  ticketId,
+  IconType
+}) => <Card
+  className='incident-card'
+  style={{ backgroundColor: event.backgroundColor }}
+>
+  <CardHeader
+    title={titleText}
+    subtitle={timeFormattedToMultipleZones(DateTime.fromISO(event.occurred ? event.occurred : event.Occurred))}
+    iconStyle={{
+      color: 'black'
+    }}
+    avatar={IconType ? <Avatar icon={<IconType />} /> : null}
+  />
+  <CardText>
```

Comment:
This could be CardActions instead of CardText. This keeps it semantic, and also the padding is more compact.